src/inputs/input/__test__/input.test.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * To run this test from the church-management/client folder, execute the following command:
+ * npx jest ./src/inputs/input/__test__/input.test.js
+ */
+
+/**
+ * NOTE: This unit test is not done for the Input, they were only introduced to test peices touched
+ * from the gap (v1 - v2).
```

Comment:
Some awkward grammar and some typos in here.  Here's a suggested rewrite:
```
NOTE: This unit test fixture does not provide full coverage for the Input component.
Only tests needed cover pieces touched from the gap (v1 - v2) have been written at this time.
```

Reply:
Thank you and Fixed.

Reply:
Oh yuck!  I just re-read this, and even my grammar is a bit awkward! ;-)
Missing the word "to" in the second sentence there, and I'm gonna tweak the rest of that sentence

> Only tests needed **_to_** cover pieces touched pieces touched ~from~ **_in closing_** the gap (v1 - v2) have been written at this time.

Any objections to one more update?

Reply:
503d94d ( on #234 ) implements the agreed upon edit.  So with that, I am satisfied.